# SOME DESCRIPTIVE TITLE.
# Copyright (C) 2023, aiogram Team
# This file is distributed under the same license as the aiogram package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2023.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: aiogram \n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2023-07-30 18:31+0300\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 2.12.1\n"

#: ../../dispatcher/errors.rst:3
msgid "Errors"
msgstr ""

#: ../../dispatcher/errors.rst:7
msgid "Handling errors"
msgstr ""

#: ../../dispatcher/errors.rst:9
msgid ""
"Is recommended way that you should use errors inside handlers using try-"
"except block, but in common cases you can use global errors handler at "
"router or dispatcher level."
msgstr ""

#: ../../dispatcher/errors.rst:12
msgid ""
"If you specify errors handler for router - it will be used for all "
"handlers inside this router."
msgstr ""

#: ../../dispatcher/errors.rst:14
msgid ""
"If you specify errors handler for dispatcher - it will be used for all "
"handlers inside all routers."
msgstr ""

#: ../../dispatcher/errors.rst:34
msgid "ErrorEvent"
msgstr ""

#: aiogram.types.error_event.ErrorEvent:1 of
msgid ""
"Internal event, should be used to receive errors while processing Updates"
" from Telegram"
msgstr ""

#: aiogram.types.error_event.ErrorEvent:3 of
msgid "Source: https://core.telegram.org/bots/api#error-event"
msgstr ""

#: ../../docstring aiogram.types.error_event.ErrorEvent.update:1 of
msgid "Received update"
msgstr ""

#: ../../docstring aiogram.types.error_event.ErrorEvent.exception:1 of
msgid "Exception"
msgstr ""

#: ../../dispatcher/errors.rst:45
msgid "Error types"
msgstr ""

#: aiogram.exceptions.AiogramError:1 of
msgid "Base exception for all aiogram errors."
msgstr ""

#: aiogram.exceptions.DetailedAiogramError:1 of
msgid "Base exception for all aiogram errors with detailed message."
msgstr ""

#: aiogram.exceptions.CallbackAnswerException:1 of
msgid "Exception for callback answer."
msgstr ""

#: aiogram.exceptions.UnsupportedKeywordArgument:1 of
msgid "Exception raised when a keyword argument is passed as filter."
msgstr ""

#: aiogram.exceptions.TelegramAPIError:1 of
msgid "Base exception for all Telegram API errors."
msgstr ""

#: aiogram.exceptions.TelegramNetworkError:1 of
msgid "Base exception for all Telegram network errors."
msgstr ""

#: aiogram.exceptions.TelegramRetryAfter:1 of
msgid "Exception raised when flood control exceeds."
msgstr ""

#: aiogram.exceptions.TelegramMigrateToChat:1 of
msgid "Exception raised when chat has been migrated to a supergroup."
msgstr ""

#: aiogram.exceptions.TelegramBadRequest:1 of
msgid "Exception raised when request is malformed."
msgstr ""

#: aiogram.exceptions.TelegramNotFound:1 of
msgid "Exception raised when chat, message, user, etc. not found."
msgstr ""

#: aiogram.exceptions.TelegramConflictError:1 of
msgid ""
"Exception raised when bot token is already used by another application in"
" polling mode."
msgstr ""

#: aiogram.exceptions.TelegramUnauthorizedError:1 of
msgid "Exception raised when bot token is invalid."
msgstr ""

#: aiogram.exceptions.TelegramForbiddenError:1 of
msgid "Exception raised when bot is kicked from chat or etc."
msgstr ""

#: aiogram.exceptions.TelegramServerError:1 of
msgid "Exception raised when Telegram server returns 5xx error."
msgstr ""

#: aiogram.exceptions.RestartingTelegram:1 of
msgid "Exception raised when Telegram server is restarting."
msgstr ""

#: aiogram.exceptions.RestartingTelegram:3 of
msgid ""
"It seems like this error is not used by Telegram anymore, but it's still "
"here for backward compatibility."
msgstr ""

#: aiogram.exceptions.RestartingTelegram:6 of
msgid ""
"Currently, you should expect that Telegram can raise RetryAfter (with "
"timeout 5 seconds)"
msgstr ""

#: aiogram.exceptions.RestartingTelegram:7 of
msgid "error instead of this one."
msgstr ""

#: aiogram.exceptions.TelegramEntityTooLarge:1 of
msgid "Exception raised when you are trying to send a file that is too large."
msgstr ""

#: aiogram.exceptions.ClientDecodeError:1 of
msgid ""
"Exception raised when client can't decode response. (Malformed response, "
"etc.)"
msgstr ""
